tests/data/linux_portactivity-v1.json

```diff
@@ -29,253 +29,253 @@
             ],
             "application_metrics": [
                 {
-                    "metric": "ssh_from_CLOSE_WAIT",
+                    "metric": "ssh.from.CLOSE_WAIT",
```

Comment:
doesnt this require changes in the graphing?

Reply:
nope, that is completely separate.  Neat huh?

Reply:
😕